src/drivers/display/fb/lv_linux_fbdev.c

```diff
@@ -249,14 +253,27 @@ void lv_linux_fbdev_set_force_refresh(lv_display_t * disp, bool enabled)
  *   STATIC FUNCTIONS
  **********************/
 
+static void write_to_fb(lv_linux_fb_t * dsc, uint32_t fb_pos, const void * data, size_t sz)
+{
+#if LV_LINUX_FBDEV_MMAP
+    uint8_t * fbp = (uint8_t *)dsc->fbp;
+    lv_memcpy(&fbp[fb_pos], data, sz);
+#else
+    if(pwrite(dsc->fbfd, data, sz, fb_pos) < 0)
+        perror("Error writing to fbfd");
```

Comment:
LV_LOG_ERROR("writie failed: %d", errno);

Reply:
Other error reporting in this file used `perror()`, that's why I did it that way.
In places where `LV_LOG_ERROR()` is used it tends to print `strerror(errno)` rather than `errno`.
I've changed it as you suggested.

Reply:
It is more appropriate to use `LV_LOG`, which is more convenient for managing log output.

Reply:
Yes, you can consider printing strerror(errno)